system-addon/bootstrap.js

```diff
@@ -78,6 +78,7 @@ function init(reason) {
 function uninit(reason) {
   if (activityStream) {
     activityStream.uninit(reason);
+    activityStream = null;
```

Comment:
This is to allow tests to clear/disable activity-stream and immediately uninstall (which resulted in back-to-back `uninit`s).